### DISCUSSION TITLE
Added "auto" action type to v4v spec

### DISCUSSION THREAD

**Discussion by daveajones** [Podcastindex-org/podcast-namespace#520] [Announcements - Changes Made]
To support periodic payments from apps to podcasts even when not listening.

Ref:  [https://github.com/Podcastindex-org/podcast-namespace/blob/main/value/value.md#appendix-b---payment-actions](https://github.com/Podcastindex-org/podcast-namespace/blob/main/value/value.md#appendix-b---payment-actions)

**Comment by dhk2** [comment]
First implementation.

The PeerTube Lightning Plug-in currently is implementing auto payments with the Patronage feature. It depends on Alby wallet authorization via API so that the server can make the payments at midnight without having to get interactive authorization. There is no quid pro quo for the patronage, it's the other half of V4V, listeners freely offering value to enable/incentivize creators to create more.

Many features are User configurable for the patronage:
![image](https://github.com/Podcastindex-org/podcast-namespace/assets/10343429/4255b0d8-11b6-41d3-b34c-c140347a8320)


Example anonymous payment:
![image](https://github.com/Podcastindex-org/podcast-namespace/assets/10343429/92de7599-7e6e-42e6-8df3-4ea6e9d9c7c4)

Example normal payment:
![image](https://github.com/Podcastindex-org/podcast-namespace/assets/10343429/e2bfbeaf-3715-47ff-aced-e559e5d5e528)

TODOs:
- make patronage levels configurable by channel. 
- keep track of failed split payments to retry, currently if any splits succeed the subscription payment is considered successful
-
